### ISSUE TITLE
Any idea how it could work with a  voron trident

### ISSUE THREAD

**Issue by a625013463** [VIN-y/MissChanger#14]
Do you have any idea how it could work with a voron trident?

may be you have an idea and some kind of picture to show how :-)

thanks and greetings

**Comment by VIN-y** [comment]
The Voron Trident use the same XY gantry as the 2.4 (it is just bolted in place); thus, the tool-change mechanism and dock should already works for it, without any modification needed. The bits that need to be changed are the XLR panels and the Nudge (calibration probe) mount.

In other words, it should be pretty easy to do. However, I can't afford (or have) a Trident to work with. 

So... either:
1. Someone, who have a Trident, pick up that tab. (most likely... hopefully)
2. The project got popular enough and get noticed by someone who is willing to sent me a Trident. (better not get your hope up)
3. I save up the funds needed for a Trident. (early-mid 2025)

**Comment by VIN-y** [comment]
Oh wait. I just check the Trident manual for the dimensions. The XLR panels should fit.

**Comment by a625013463** [comment]
Thanks a lot, I own a Voron Trident and I'll go back and try it out, thanks again

**Comment by Marck80** [comment]
Were you able to mount it on the Trident?

**Comment by VIN-y** [comment]
@Marck80 At the bottom. Pretty much the same place where they would be on the 2.4.

**Comment by VIN-y** [comment]
I got the chance to play with a couple of Trident at SMMRF2024. From which I got 2 observations:
1. The Nudge probe can be used with the Trident using the float base that is provided in the original repo. When you don't need it, you can just leave it to the side. Because the gantry doesn't go down, there is a lot of space to play with.
2. The XLR panels should work on the bigger Trident, 350. But, things will get dicey for smaller ones. As the z drag chain terminated where the foot of the panel is meant to stick out and be bolted to the extrusion at the back.

**Comment by Marck80** [comment]
I just have the 300..I'll try to print and see if there are any problems..I'll let you know

**Comment by Battlecake91** [comment]
Is there any update on this topic?

I'm building a Trident 350 with more on the Z axis (350x350x400) and I 100% want a toolchanger system for stealthburner (as I'm using a lot of adittional stuff there - so DAKSH isn't a good option for me).

I will try to integrate this system on my Trident, but if you saw some bigger problems, it could help me to save some time.

**Comment by VIN-y** [comment]
> Is there any update on this topic?

We got some one who is building MissChanger on a Trident. From their feedback (which has been implemented to v1.2):
- The top bar of the dock has to be removed.

With that said, no further compatibility issue has been found thus far.

Nevertheless, I have decided to extend the alpha to see how that Trident build go. I cannot publicly give you a date on when it is ready, though. As is the nature of an open-source project (where time is donated), it is done when it is done.

**Comment by VIN-y** [comment]
yes. I usually gave everyone a head-sup on discord whenever I about to do something.

**Comment by VIN-y** [comment]
FYI.
Early tester has confirmed the need for a top-hat. The umbilical rubbing on the top panel will cause problems with probe accuracy.

**Comment by ArtytheSecond2nd** [comment]
Any updates here? It seems almost complete but isn't listed as supported yet.

**Comment by VIN-y** [comment]
> Any updates here? It seems almost complete but isn't listed as supported yet.

Oh right. I have been sketched out by Github's AI adoption and the site reliability issues that follows, and totally forgot about it.

Anyway, have a look at MCv2 project I have put together on Gitlab: https://gitlab.com/misschanger/misschanger-v2. The tool-changer is built on and for Trident.

The frontpage has been updated accordingly.

---

## 260712. **MCv2**, A new version of MissChanger released on Gitlab and Printable

The assembly guide and configuration is free and available in the [Gitlab wiki](https://gitlab.com/misschanger/misschanger-v2/-/wikis/home). For the full project description, have a look at the [Gitlab frontpage](https://gitlab.com/misschanger/misschanger-v2) or the [Printable description](https://www.printables.com/model/1778493-misschanger-v2).

While the CAD file can be free or paid, depending on your needs, on Printable.

| Price  | Include                       | Link                                                         |
| ------ | ----------------------------- | ------------------------------------------------------------ |
| $0.00  | .3mf                          | [MissChanger v2 \| Printables.com](https://www.printables.com/model/1533585-misschanger-v2) |
| $25.00 | .step<br>native FreeCAD files | [MissChanger v2 \| Printables.com](https://www.printables.com/model/1778493-misschanger-v2) |
